openscm/core/time.py

```diff
@@ -140,8 +140,8 @@ def _format_datetime(dts: np.ndarray) -> np.ndarray:
         return np.array([], dtype="datetime64[s]")
 
     dtype = dts.dtype.type
-    if issubclass(dtype, np.object):
-        dtype = np.dtype(type(dts[0])).type
+    # if issubclass(dtype, np.object):
```

Comment:
Let's please leave it like before. Getting only the first element type should only be a fallback option.